dev/kyuubi-extension-spark-3-1/src/test/scala/org/apache/spark/sql/KyuubiExtensionSuite.scala

```diff
@@ -418,4 +418,183 @@ class KyuubiExtensionSuite extends QueryTest with SQLTestUtils with AdaptiveSpar
       }
     }
   }
+
+  test("get simple name for DDL") {
+
+    import scala.collection.mutable.Set
```

Comment:
move up?

Reply:
this test is used for getting simple name for myself...
i use those data set into #{sql-classification-default.json}